### DISCUSSION TITLE
I always want to turn off "random-UUID".

### DISCUSSION THREAD

**Discussion by yosang0511** [XorTroll/emuiibo#246] [Q&A]
I always want to turn off "random-UUID".
Please teach me the way.
And I can't find amiibo.json.

**Comment by lookye** [comment]
Please search help on Discord. Its linked on top in the readme.

**Comment by impeeza** [comment]
you can read the readme.md of the repo and the release of the 1.0.0 version.  is clearly explained there.

with the latest overlay is as simple as press ZL/ZR.

as stated before if you need help go to the proper site, like Discord or GBATemp.
